.github/workflows/test.yml

```diff
@@ -36,10 +36,16 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install hatch
-    - name: Run Tests
+    - name: Run Tests (with zarr)
+      if: matrix.zarr-version != 'none'
       run: |
         hatch run test.py${{ matrix.python-version }}-${{ matrix.zarr-version }}:list-env
         hatch run test.py${{ matrix.python-version }}-${{ matrix.zarr-version }}:test-cov
+    - name: Run Tests (without zarr)
+      if: matrix.zarr-version == 'none'
+      run: |
+        hatch run test-base.py${{ matrix.python-version }}:list-env
+        hatch run test-base.py${{ matrix.python-version }}:test-cov
```

Comment:
@d-v-b, let me know if you have a better idea for doing the hatch matrix here... or if you're happy to switch this repo over to using `uv` and do it similarly to https://github.com/ome-zarr-models/ome-zarr-models-py/pull/280

Reply:
but that can be in a later PR